tests/module/test_lqr.py

```diff
@@ -118,7 +118,7 @@ def test_lqr_ltv(self, device='cpu'):
         Q = torch.tile(torch.eye(n_state + n_ctrl, device=device), (n_batch, 1, 1))
         p = torch.tensor([[-1.00, -0.68, -0.35, -1.42, 0.23, -1.73, -0.54],
                           [-1.00, -0.68, -0.35, -1.42, 0.23, -1.73, -0.54]], device=device)
-        rt = torch.arange(1, T+1).view(T, 1, 1)
+        rt = torch.arange(1, T+1).view(T, 1, 1).to(device)
```

Comment:
This is more efficient
`rt = torch.arange(1, T+1, device=device).view(T, 1, 1)`